### PR TITLE
4. View a list of all reviews

### PR DESCRIPTION
**Pull request by panicroll**

### CODE REVIEW

File: src/components/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    return (
+        
+        <ul className="reviews_list">
+            {reviews.map((review) => {
+                return (
+                   <li className="review_card" key={review.review_id}>
+                    <h2>{review.title}</h2>
+                    <p>{review.category}</p>
+                    <p>{review.owner}</p>
+                    <p>{review.review_body}</p>
```
Comment (dappernerddesigns) [inline src/components/Reviews.jsx]: The BE ticket for this should not have the body returned. You're returning the who review on the list. That is a lot of text for someone to scroll past before finding the next one. This should be removed.

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
This has a good start but needs some amendments. On a mobile screen the container is off the screen with a lot of the text and information. Currently this application is unusable on a mobile device.

There is a comment for this, the body on your review cards should not be showing. Putting the entire review on this page makes it extremely long for the user.

Take a look at the points outlined and then resubmit please.

File: src/components/Reviews.jsx (hunk 1 above)
Comment (panicroll) [reply src/components/Reviews.jsx]: Removed review body, thank you.

**Review by dappernerddesigns — APPROVED** [review]
Much better! Well done. Merge away and move on when ready.

**Pull request merged**